demo/code-demo/scripts/session_utils.js

```diff
@@ -0,0 +1,59 @@
+var _session = {};
+
+function SetSession(key, value) {
+    // This is a temporal solution for session (which is cleared on reload). Should be replaced with a real session.
+    _session[key] = value;
+}
+
+function GetSession(key) {
+    // This is a temporal solution for session (which is cleared on reload). Should be replaced with a real session.
+    return _session[key];
+}
+
+function SetSessions(accessToken, embedReportUrl, embedReprotId) {
+    SetSession("accessToken", accessToken);
+    SetSession("embedUrl", embedReportUrl);
+    SetSession("embedId", embedReprotId);
+}
+
+function UpdateSessions() {
+    var txtAccessToken = $('#txtAccessToken').val();
+    if (txtAccessToken)
```

Comment:
what happens if these strings are empty strings? do you want to update the session with them anyway?